build/ci.go

```diff
@@ -357,8 +357,10 @@ func doLint(cmdline []string) {
 		packages = flag.CommandLine.Args()
 	}
 	// Get metalinter and install all supported linters
-	build.MustRun(goTool("get", "gopkg.in/alecthomas/gometalinter.v2"))
-	build.MustRunCommand(filepath.Join(GOBIN, "gometalinter.v2"), "--install")
+	os.Setenv("GO111MODULE", "on")
+	build.MustRun(goTool("get", "gopkg.in/alecthomas/kingpin.v3-unstable@63abe20a23e29e80bbef8089bd3dee3ac25e5306"))
```

Comment:
What's the reason for this version bump?

Reply:
I like to use up to date software.

Reply:
lol. weird.